Parsers/ASimAuthentication/Parsers/ASimAuthenticationM365Defender.yaml

```diff
@@ -15,6 +15,7 @@ References:
 Description: |
   This ASIM parser supports normalizing endpoint authentication events, collected by Microsoft 365 Defender for Endpoint, stored in the  DeviceLogonEvents table, to the ASIM Authentication schema.
 ParserName: ASimAuthenticationM365Defender
+EquivalentBuiltInParser: _ASim_Authentication_M365Defender
```

Comment:
@KustoKing Are these changes due to positions changing, is there anything added new or got removed?
Other than EquivalentBuiltInParser property?

Reply:
Items sorted